### ISSUE TITLE
Usage clarification of isContaminant and isNotContaminant

### ISSUE THREAD

**Issue by Ziggsblitz** [benjjneb/decontam#63]
I was hoping to get some clarification regarding correct/appropriate use of your tool so that I am using it as it was intended. In particular, regarding the isContaminant and isNotContaminant functions and their respective readouts for the prevalence based method.

Please correct me if I'm wrong or misunderstanding the tool as I describe it below:

isContaminant should be used with medium to high microbial biomass samples. This method begins with the assumption that OTUs/ASVs are legitimate signals and that prevalence in a higher proportion of negative controls over biological samples signifies a particular OTU is a contaminant. In these samples, running isContaminant with blank extraction (negative) controls should result in p.prev scores that, based on a user-designated threshold (default=0.5), would identify OTUs that are contaminants, as communicated by the "contaminant" column and values of TRUE in the output. A value of TRUE, in this case, indicates that a particular OTU had a p.prev score > 0.5 and that OTU is considered a contaminant.

Conversely, isNotContaminant should be used with low to no (expected) microbial biomass samples. This method begins with the assumption that all OTUs/ASVs are contaminants and that those that occur in a greater proportion of biological samples over negative controls are likely legitimate signals.  In these cases, running isNotContaminant with negative controls results in p.prev scores that, based on a user-designated threshold, would identify OTUs that are NOT contaminants (legitimate signals), as communicated by the "not.contaminant" column and values of TRUE in the output. A value of TRUE, in this case, indicates that a particular OTU had a p.prev score > 0.5 and that OTU should be considered legitimate.

Thank you for your time!

**Comment by benjjneb** [comment]
Your conceptual description all reads right to me. The following detailed bits should be changes, though. The following:

> A value of TRUE, in this case, indicates that a particular OTU had a p.prev score > 0.5

should instead read (in both places)

"A value of TRUE, in this case, indicates that a particular OTU had a p.prev score < 0.5"

and the default threshold is 0.1 not 0.5.

**Comment by Ziggsblitz** [comment]
Thank you for clarifying, and I apologize for having those backwards. 

To clarify the default threshold is 0.1 for isContaminant and 0.5 for isNotContaminant, but thresholds should still be evaluated on a case by case basis based on distribution and biological relevance?

**Comment by benjjneb** [comment]
> but thresholds should still be evaluated on a case by case basis based on distribution and biological relevance?

Yep, this is highly recommended over just accepting the defaults blindly.
